staff/emma-cowdery/precourse/Tema-1/README.md

```diff
@@ -0,0 +1,17 @@
+# CALCULADORA
+
+![calculator logo](calculator-icon.png)
+
+## - INTRODUCCIÓN
+
+Calculadora que da el resultado de la suma, resta, multiplicación y diviión de dos numeros dados.
+
+## - COMO USAR
+
+Al llamar la función (*calculate*) introduces los dos numeros deseados en ella.
+
+### - **Ejemplo:**
```

Comment:
The `**` aren't necessary in this case because it is a heading 3 (bold style is include)